make/photon/prepare/utils/configs.py

```diff
@@ -354,6 +356,11 @@ def parse_yaml_config(config_file_path, with_trivy):
 
     return config_dict
 
+def get_redis_schema(redis=None):
+    if 'tlsOptions' in redis and redis['tlsOptions']['rootCA'] is not None:
```

Comment:
If rootCA is not exist in the redis['tlsOptions'], it will get KeyError: 'rootCA', you should use redis['tlsOptions'].get('rootCA') is not None to check its value.